src/panels/lovelace/hui-root.js

```diff
@@ -138,13 +146,39 @@ class HUIRoot extends NavigateMixin(EventsMixin(PolymerElement)) {
         type: Object,
         observer: '_routeChanged'
       },
-      routeData: Object
+
+      _notificationsOpen: {
+        type: Boolean,
+        value: false,
+      },
+
+      routeData: Object,
     };
   }
 
   constructor() {
     super();
     this._debouncedConfigChanged = debounce(() => this._selectView(this._curView), 100);
+
+    this._debouncedNotificationsClose = debounce(() => {
+      this._notificationsOpen = false;
+    }, 100);
+
+    this._debouncedNotificationsOpen = debounce(() => {
+      this._notificationsOpen = true;
+    }, 100);
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.addEventListener('close-notification-drawer', this._debouncedNotificationsClose);
```

Comment:
This is overkill. Just have the button use a 2-way databinding to the notificationDrawer open ?